andromeda-ui/components/LatLonCoverNote.tsx

```diff
@@ -0,0 +1,20 @@
+import Anchor from "../components/Anchor";
+
+export default function LatLonCoverNote() {
+    return <p className="text-sm mt-4 max-w-prose" >
+        Note:
+        The landcover satellite data is only available for the contiguous United States 
+        from 2022 and is generated using&nbsp;
```

Comment:
[It looks ](https://www.nass.usda.gov/Research_and_Science/Cropland/sarsfaqs2.php#:~:text=The%202022%20CDL%20product%20was%20released%20January%2030%2C%202023)like the Cropland Data Layer for CropScape is updated regularly early in the new year for the previous year.

I imagine LatLonCover will use updated data when it's released. Could '2022' be dynamically linked somehow to the state of the data used in LatLonCover to reduce manual overhead for keeping notes up-to-date? Or a more general statement about using the CDL version with LatLonCover?

Reply:
In the version of LatLonCover we are including the year hard coded to 2022. Could you create an issue in LatLonCover for a way to inquire about the default year (or possibly just make it a argument passed in to the function)?